src/components/shared/MarkerTooltipContents.js

```diff
@@ -497,9 +497,9 @@ function getMarkerDetails(
               timings.initial_state + ' – ' + timings.final_state
             )}
             {triggers}
-            {timings.page_faults === undefined
-              ? null
-              : _markerDetail('gcfaults', 'Page faults', timings.page_faults)}
+            {timings.page_faults
```

Comment:
You're the one deciding here, but I still want to ask the question to make sure that it's what you want: if the value is `0` it won't be displayed, is that what you want ?

Reply:
for page_faults, yes.  For others. not always,.  I'll check them.